Documentation/markdown/GitProcess.md

```diff
@@ -0,0 +1,17 @@
+# Git Workflow
+
+![](./fig/Git_Workflow.png)
+
+Here you will find description about Git Workflow you need working with OpenInfraPlatform
+
+## 1. Clone the repository
+
+1.1. Go to OpenInfraPlatform on [GitHub](https://github.com/tumcms/Open-Infra-Platform).
+
+1.2. Login to your account and *Fork* OpenInfraPlatform to your account.
+
+1.3. Select green button *Clone or download* and copy url, which will shows up.
+
+1.4. Open your File Explorer and select the folder, where you want to clone the OpenInfraPlatform (e.g. C:\dev). Right mouse click in this folder and select *Git Bash Here*.
+
+1.5. In the window, which you have opened, in the command line write down *git clone* and paste link you have just copied. This will start a download.
```

Comment:
Write out the command *git clone https://github.com/<your_username>/Open-Infra-Platform.git*

Reply:
Done